### ISSUE TITLE
Letter C - Brazil

### ISSUE THREAD

**Issue by lmanalas** [pop-os/pop#576]
**Distribution (run `cat /etc/os-release`):**
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**

So, first of all, thank you guys for the excellent OS, BUT I'm from Brazil and we have this strange letter 'ç' (cedilha/cedilla), my keyboard is a ducky shine 5, the key layout is 'english international' or something, the international one. I was unable to type 'ç' like I was in windows and after some time searching I found this guide (https://superuser.com/questions/1075992/cedilla-under-c-%C3%A7-in-us-international-with-dead-keys-keyboard-layout-in-linu), I did it and it worked, so II was hoping you guys could add it in a future update or something
-@tasogareriiku

**Steps to reproduce (if you know):**

**Expected behavior:**

**Other Notes:**

**Comment by davidbanhos** [comment]
I've been working on _Ubuntu-like_ distros for awhile and I always executed the script bellow in order to have "ç" working as " **' + c** "

Hope it helps. 

[https://github.com/silveira/cedilha.sh](url)

**Comment by hugows** [comment]
Funcionou perfeitamente bem no 20.04.  Mesmo com notebook ABNT2 eu prefiro usar o layout americano...
(Ah o link acima não está correto)

**Comment by raelgc** [comment]
Take a look on these instructions have I've wrote for several distros (I've even created a Ubuntu package): https://github.com/raelgc/win_us_intl/

**Comment by adriano-moreira** [comment]
This is not a issue, no script required

the correct way to use is:

configure your keyboard to any English(Intl)

use with `AltGr + ,` to `ç`
use with `AltGr + Shift + ,` to `Ç`
AltGr is the rigth side Alt

tested on Pop!OS 21.10, now


![image](https://user-images.githubusercontent.com/3504273/147507215-7b978971-9374-4ce6-853e-f67bf33a3062.png)

**Comment by aylons** [comment]
I disagree, @adriano-moreira, this is an issue and your proposal is just a workaround instead of a proper adaptation.

Deadkey with acute is how cedilla is written on keyboards by any Portuguese speaking person in any system. People are used to write cedilla by using the deadkey with acute and not only they do not want to change their habits to fit a foreign pattern, and an accord with AltGr is less comfortable and slower.

**Comment by hugows** [comment]
Yes, I type portuguese-on-ANSI-keyboards on Windows, Mac and Linux always with this method (deadkeys) for 20 years now. No way I can learn a "proper way".
